src/components/Input/index.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { letterSpacing, colors, size } from '../../utils'
+
+const { black, gray3 } = colors
+
+const Input = styled.input`
+  width: 100%;
+  padding: 1rem;
+  box-sizing: border-box;
+  border: 1px solid rgba(224, 224, 224, 0.5);
+  background: rgba(196, 196, 196, 0.05);
+  outline: none;
```

Comment:
@jema28 Im thinking we can add some outline here, with primary color? and make it a prop maybe